features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularContext.cpp

```diff
@@ -69,11 +69,20 @@ ControlPlane_netif *QUECTEL_BG96_CellularContext::get_cp_netif()
 
 nsapi_error_t QUECTEL_BG96_CellularContext::do_user_authentication()
 {
+    uint8_t type = 1;
+    if ((uint8_t)_pdp_type > 1) {
+        type = 2;
+    }
     if (_pwd && _uname) {
-        if (_at.at_cmd_discard("+QICSGP", "=", "%d%d%s%s%s%d", _cid, 1,
+        if (_at.at_cmd_discard("+QICSGP", "=", "%d%d%s%s%s%d", _cid, type,
```

Comment:
If `context_type` can be only IPV4 or IPV6 then we should disable IPV4V6 PDPs completely?